tests/conftest.py

```diff
@@ -248,13 +248,17 @@ def real_chromadb():
     yield
     chromadb_patcher.start()
 
+def _start_forked_server(repo):
+    if os.fork() != 0: 
+        return 
+    start_server(repo) 
 
 @pytest.fixture(name="start_server")
 def _start_server(repo):
     def _start():
         context = multiprocessing.get_context("spawn")
         server_process = context.Process(
-            target=start_server, args=(repo.working_dir,), daemon=True
+            target=_start_forked_server, args=(repo.working_dir,), daemon=True
```

Comment:
I have modified _start_server from tests/conftest.py file to detach the server process from the test process. If I don't do that wait function hangs on because it waits for the server process (which is also the same as the test) to terminate.

Reply:
let's see if this method works on all tests and all platforms currently enabled!

Reply:
I have enabled the pipeline so now we should know

Reply:
looks like this fix only worked on Linux, it fails on

- windows https://github.com/kantord/SeaGOAT/actions/runs/6372185774/job/17295086591?pr=284
- mac os https://github.com/kantord/SeaGOAT/actions/runs/6372185776/job/17295086696?pr=284

Reply:
Interesting 🤔. I'm going to have a look on this.

Side note : I'm on macos and it works.